elections/2022/TOC/results.md

```diff
@@ -0,0 +1,26 @@
+# 2022 CFF Technical Oversight Committee Election Results
+
+Many thanks to all the nominees and everyone who voted in the election. 
+
+I’m pleased to announce our new TOC members will be:
+
+* Ruben Koster (VMware)
+* Beyhan Veli (SAP)
+* Andy Hunt (GOV.UK)
+
+They'll be joining Eric Malm (VMware) and Stephan Merker (SAP), whose seats will be up for election next year.
+
+Congratulations to Ruben, Beyhan and Andy!
+
+## Details
+
+Actual votes cast: 60
+
+Complete results of the election, using the Schulze rules for Condorcet completion, are:
+
+1. Ruben Koster
+2. Beyhan Veli
+3. Andy Hunt
+4. Peter Burkholder
+
+Records of all cast ballots are available in [CSV format here](ballots.csv).
```

Comment:
I'm going to change this to the absolute link to GitHub so that the rendered page on Elekto will also link to the CSV.
```suggestion
Records of all cast ballots are available in [CSV format here](https://github.com/cloudfoundry/community/blob/main/elections/2022/TOC/ballots.csv).
```